src/main-client.ts

```diff
@@ -951,6 +1430,26 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('api/v3/myTrades', params);
   }
 
+  getOrderRateLimit(): Promise<OrderCountUsageResponse> {
```

Comment:
```suggestion
  getOrderRateLimitUsage(): Promise<OrderRateLimitUsage[]> {
```

See comment about OrderCountUsageResponse in types/spot.ts.